security-force-monitor/sfm-cms
# Disallow crawling by Bingbot and PetalBot

## Overview

Connects #722

## Testing Instructions

* Deploy to production, then verify that the new `robots.txt` directive successfully blocks Bing and Petal bots here: https://technicalseo.com/tools/robots-txt/
